README.md

```diff
@@ -36,10 +40,11 @@ You should have the following ready before beginning with any board:
     * Open Boards Manager from Tools > Board menu and install esp8266 platform 2.1.0 or later 
     * Select your ESP8266 board from Tools > Board menu after installation
     
-2. Open the AzureIoTHub ESP8266 sample from the Arduino IDE File->Examples menu.
```

Comment:
> esp8266 platform 2.1.0

Not sure if we need 2.2.0. I need https://github.com/arduino-libraries/AzureIoT/pull/13/commits/aa163f45d3fa628a8cd6c1a38e65e8eb86dc8c07 to get #13 building.

I remember needing to change ``fprintf`` when testing because of https://github.com/esp8266/Arduino/commit/4c587e28bc188efc852039606cac641768323ee3.